addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ReplacerParam.java

```diff
@@ -108,6 +112,22 @@ public class ReplacerParam extends VersionedAbstractParam {
                             true,
                             REPORT_TO_REPLACEMENT,
                             List.of(),
+                            false),
+                    new ReplacerParamRule(
+                            MODIFIED_SINCE_DESC,
+                            ReplacerParamRule.MatchType.REQ_HEADER,
+                            HttpHeader.IF_MODIFIED_SINCE,
```

Comment:
Could we publish an up-to-date commonlib to maven? Then I could depend on it, and use HttpFieldsNames. Or is it better for some reason to depend on the add-on, can that be done non-optionally?

Reply:
You can already depend/use it. Are you getting an error?

Reply:
No, I hadn't tried yet. Wanted to ask about this first.

Reply:
Hmmm seems it's already depending on it as an add-on. It wouldn't resolve the class the other day though. I'll do some refreshes and see if its just a dumb Eclipse/Windows thing 😞